sass/_vars.scss

```
@@ -54,9 +54,11 @@ $card-list-gap: 15px;
 // CSS Vars
 :root {
     --header-height: 60px;
+    scroll-padding-top: 76px;
```

I would define this as a CSS var, which is based on `--header-height` (no need to add second definition for _landscape-up_ breakpoint):

```suggestion
    --scroll-padding-top: calc(var(--header-height) + 16px);
```

---

Oops. The bits in `_vars.scss` didn't get cleaned up when I put the properties in the proper place in `_html.scss`.

Good idea to add a variable though. I'll fix that up.